src/components/TextField/TextField.tsx

```diff
@@ -29,6 +29,10 @@ export const TextField: React.FC<Props> = ({
   const [touched, setTouched] = useState(false);
   const hasError = touched && required && !value;
```

Comment:
```suggestion
  const hasError = touched && required && !value.trim();
```
filled all inputs only with empty space.
any errors visible, but add button still disabled.

better to trim value for hasError variable.
![Screenshot 2024-04-15 at 23 39 27](https://github.com/mate-academy/react_movies-list-add-form/assets/33029864/c4532066-46b4-4d78-b3b9-b85f84e6d9e4)